### ISSUE TITLE
Error in melee combat.

### ISSUE THREAD

**Issue by jinlan** [NoImageAvailable/CombatExtended#630]
```
Added injury to Head but it should be impossible to hit it. pawn=AA_Nightling43325 dinfo=(def=Scratch, amount= 9, instigator=Wolf_Timber43380, angle=315.0)
Verse.Log:Error(String, Boolean)
Verse.Hediff_Injury:PostAdd(Nullable`1)
Verse.HediffSet:AddDirect(Hediff, Nullable`1, DamageResult)
Verse.Pawn_HealthTracker:AddHediff(Hediff, BodyPartRecord, Nullable`1, DamageResult)
Verse.DamageWorker_AddInjury:FinalizeAndAddInjury_Patch1(Object, Pawn, Hediff_Injury, DamageInfo, DamageResult)
Verse.DamageWorker_AddInjury:FinalizeAndAddInjury(Pawn, Single, DamageInfo, DamageResult)
Verse.DamageWorker_Scratch:ApplySpecialEffectsToPart(Pawn, Single, DamageInfo, DamageResult)
Verse.DamageWorker_AddInjury:ApplyDamageToPart_Patch1(Object, DamageInfo, Pawn, DamageResult)
Verse.DamageWorker_AddInjury:ApplyToPawn(DamageInfo, Pawn)
Verse.DamageWorker_AddInjury:Apply(DamageInfo, Thing)
Verse.Thing:TakeDamage_Patch1(Object, DamageInfo)
CombatExtended.Verb_MeleeAttackCE:ApplyMeleeDamageToTarget(LocalTargetInfo)
CombatExtended.Verb_MeleeAttackCE:TryCastShot_Patch1(Object)
Verse.Verb:TryCastNextBurstShot_Patch0(Object)
Verse.Verb:WarmupComplete()
Verse.Verb:TryStartCastOn_Patch2(Object, LocalTargetInfo, Boolean, Boolean)
RimWorld.Pawn_MeleeVerbs:TryMeleeAttack_Patch1(Object, Thing, Verb, Boolean)
RimWorld.<MakeNewToils>c__AnonStorey1:<>m__3()
Verse.AI.<FollowAndMeleeAttack>c__AnonStorey3:<>m__0()
Verse.AI.JobDriver:DriverTick()
Verse.AI.Pawn_JobTracker:JobTrackerTick()
Verse.Pawn:Tick_Patch2(Object)
Verse.TickList:Tick()
Verse.TickManager:DoSingleTick()
Verse.TickManager:TickManagerUpdate()
Verse.Game:UpdatePlay()
Verse.Root_Play:Update()
```

I'm still not sure whether it's an CE bug. This error comes out when an "alpha animal" Nightling is attacking a wolf. And it's likely that this error only come out when head is hit ( not sure, need more experience ).

Got second error.  A bear hit a deer's head.

**Comment by NoImageAvailable** [comment]
This is overzealous vanilla logging. Error is safe to ignore, fix is to suppress it via Harmony.

**Comment by LockdownX7** [comment]
Should be fixed as of 734ecefe7b15bd1f8948236dca9f8a3a612882de.
